index.html

```diff
@@ -123,61 +123,66 @@
     </div>
 </div>
 
-<script
-src="https://code.jquery.com/jquery-3.2.1.min.js"
-integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-crossorigin="anonymous"></script>
-<script src="jquery-3.2.1.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jsrender/0.9.84/jsrender.js"></script>
 
 <script>
-    $(function () {
-
-        var renderTemplate = function (data) {
-
-            $.when($.get('password_template.html'))
-                .done(function (tmplData) {
-                    $.templates({tmpl: tmplData});
+    // https://github.com/sindresorhus/pupa
+    var pupaMini = function (tpl, data) {
+        var re = /{(.*?)}/g;
+
+        return tpl.replace(re, function (_, key) {
+            return key.split('.').reduce(function (data, prop) {
```

Comment:
You're right, no need a fancy lib to replace a few fixed keywords. We could simplify this further, since we don't have nested replacement/translation keys no need for splitting, we can just directly return 

    return data ? data[prop] : '';